react/error.tsx

```diff
@@ -10,15 +11,27 @@ import ErrorImg from './images/error-img.png'
 
 import style from './error.css'
 import { renderReadyPromise } from '.'
+import { isAdmin } from './utils/isAdmin'
 
 class ErrorPage extends Component {
   public state = { enabled: false }
-  private splunk = 0
 
   public componentDidMount() {
     window.setTimeout(() => {
       this.setState({ enabled: true })
     }, 5000)
+
+    if (isAdmin()) {
+      if (window.__ERROR__) {
+        captureException(window.__ERROR__)
+      } else if (window.__REQUEST_ID__) {
+        captureException(window.__REQUEST_ID__)
+      } else {
+        captureException(
+          'Render Runtime renderered an error page and there is no error or request id available'
+        )
+      }
+    }
```

Comment:
LGTM. 
Leaving a suggestion to improve readability:
 - invert `if` logic for early return
 - add variables for `__ERROR__` and `__REQUEST_ID__ `to minimize access (could also do it by destructuring assignment if you like)
 - fixed typo `renderered` in `defaultMessage`

```suggestion
  if (!isAdmin()) return;

  const error = window.__ERROR__;
  const requestId = window.__REQUEST_ID__;
  const defaultMessage =
    'Render Runtime rendered an error page, and there is no error or request ID available';

  if (error) {
    captureException(error);
  } else if (requestId) {
    captureException(requestId);
  } else {
    captureException(defaultMessage);
  }
 
```

Reply:
Much better @vmourac-vtex ! Thanks! I'm applying these changes.